src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -235,7 +228,7 @@ public LogUnitServer(ServerContext serverContext) {
             // support sparse files.  If we use the default 2GB file size, then
             // every time that a sparse file is closed, the OS will always
             // write 2GB of data to disk.  {sadpanda}  Use this static class
-            // var to signal to RollingLog to use a smaller file size.
+            // var to signal to StreamLogFiles to use a smaller file size.
             maxLogFileSize = 4_000_000;
         }
 
```

Comment:
Uh, not sure about HFS+, but on most file systems, writing 512B to a 2GB sparse file results in a 2GB file being allocated but only 512B being written to disk (well, plus overhead, but definitely NOT 2GB)...